app/controllers/admins_controller.rb

```diff
@@ -13,12 +13,12 @@ def show
   def items_recent_activity
     authorize! :admin_dashboard
     @days_limit = params.fetch(:days, 7).to_i
-    @events = Event.work_events.includes(:eventable).where('created_at > ?', @days_limit.days.ago)
+    @items = Event.work_events.where('created_at > ?', @days_limit.days.ago).pluck(:eventable_id).uniq
   end
 
   def collections_recent_activity
     authorize! :admin_dashboard
     @days_limit = params.fetch(:days, 7).to_i
-    @events = Event.collection_events.includes(:eventable).where('created_at > ?', @days_limit.days.ago)
+    @collections = Event.collection_events.where('created_at > ?', @days_limit.days.ago).pluck(:eventable_id).uniq
   end
```

Comment:
I tried for a while and couldn't get a working SQL query that did a distinct on the `eventable_id` and returned what we needed.  If it's possible, this would be preferable to what I am doing here, which is just fetching the `eventable_ids` and then uniquing the array, since this requires loading the items/collections later in the compontent (which triggered a bunch of changes as seen above).

Reply:
Is it possible to just look at updated_at on the collection version? I don't think we have any  events that don't update the object? That would certainly make for a simpler query.